lib/ansible/plugins/lookup/aws_kms_decrypt.py

```diff
@@ -0,0 +1,217 @@
+"""
+(c) 2018, Achintha Gunasekara <contact@achinthagunasekara.com>.
+Lookup plugin to handle decrypting of items with KMS.
+GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt).
+"""
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+import base64
+from os.path import basename
+from ansible.errors import AnsibleError
+from ansible.plugins.lookup import LookupBase
+
+
+DOCUMENTATION = """
+lookup: :aws_kms_decrypt
+author:
+    - Achintha Gunasekara <contact@achinthagunasekara.com>
```

Comment:
```suggestion
    - Achintha Gunasekara (@achinthagunasekara) <contact@achinthagunasekara.com>
```

Reply:
@gundalow I made this change on both files. Thanks.